rotkehlchen/premium/sync.py

```diff
@@ -154,7 +161,7 @@ def maybe_upload_data_to_server(self, force_upload: bool = False) -> bool:
                     f'upload to server stopped -- remote db({metadata.last_modify_ts}) '
                     f'more recent than local({our_last_write_ts})',
                 )
-                return False
+                return False, 'Remote db is more recent than local'
```

Comment:
Consistency with the other messages

```suggestion
                return False, 'Remote database is more recent than local'
```